src/containers/Create/index.jsx

```diff
@@ -1,30 +1,31 @@
 /**
  * Created by 包俊 on 2018/8/14.
  */
-import React from "react";
-import TopNav from "../../components/TopNav/index";
-import SupplyInput from "../../components/Input/index";
-import NameInput from "../../components/Input/index";
-import DecimalInput from "../../components/Input/index";
-import SymbolInput from "../../components/Input/index";
-import { deploy } from "../../contracts/tokenStore";
-import Button from "../../components/Button/index";
+import React from 'react';
+import TopNav from '../../components/TopNav/index';
```

Comment:
all `/index` can be omitted, if `require` a dir, it will look for `index.js` automatically